ui/desktop/.eslintrc.js

```diff
@@ -36,6 +38,7 @@ module.exports = {
       files: [
         './.eslintrc.js',
         './.prettierrc.js',
+        './.stylelintrc.js',
         './.template-lintrc.js',
         '.stylelintrc.js',
```

Comment:
Isn't line 41 and line 43 a duplicate? perhaps we should choose one and delete the other one?

Reply:
@calcaide good catch, fix incoming